include/deal.II/matrix_free/hanging_nodes_internal.h

```diff
@@ -922,18 +912,21 @@ namespace internal
 
     template <int dim>
     inline void
-    HangingNodes<dim>::rotate_face(
-      int                                   times,
-      unsigned int                          n_dofs_1d,
+    HangingNodes<dim>::orient_face(
+      const types::geometric_orientation    combined_orientation,
+      const unsigned int                    n_dofs_1d,
       std::vector<types::global_dof_index> &dofs) const
     {
-      const unsigned int rot_mapping[4] = {2, 0, 3, 1};
+      const auto [orientation, rotation, flip] =
+        internal::split_face_orientation(combined_orientation);
+      const int n_rotations = int(rotation) + 2 * int(flip);
 
-      times = times % 4;
-      times = times < 0 ? times + 4 : times;
+      const unsigned int rot_mapping[4] = {2, 0, 3, 1};
+      const unsigned int n_int          = n_dofs_1d - 2;
```

Comment:
What does that variable name mean? Perhaps choose something that is easier to understand?

Reply:
It is the number of dofs per line (i.e., without the two vertex dofs). I'll do a little more QA here and pick a better name.